src/main/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionUpdateActionUtils.java

```diff
@@ -20,20 +29,139 @@
      *
      * @param oldAttributeDefinition        the attribute definition which should be updated.
      * @param newAttributeDefinitionDraft   the attribute definition draft where we get the new fields.
-     * @param syncOptions                   responsible for supplying the sync options to the sync utility method.
-     *                                      It is used for triggering the error callback within the utility, in case of
-     *                                      errors.
      * @return A list with the update actions or an empty list if the attribute definition fields are identical.
      *
      */
     @Nonnull
     public static List<UpdateAction<ProductType>> buildActions(
         @Nonnull final AttributeDefinition oldAttributeDefinition,
-        @Nonnull final AttributeDefinitionDraft newAttributeDefinitionDraft,
-        @Nonnull final ProductTypeSyncOptions syncOptions) {
+        @Nonnull final AttributeDefinitionDraft newAttributeDefinitionDraft) {
+
+        return Stream.of(
+            buildChangeLabelUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
+            buildSetInputTipUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
+            buildChangeIsSearchableUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
+            buildChangeInputHintUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft),
+            buildChangeAttributeConstraintUpdateAction(oldAttributeDefinition, newAttributeDefinitionDraft)
```

Comment:
are enums actions going to be added here too?

Reply:
No, they have its own class. Thus, it's easier to test only the enums separated from the attribute definitions unit tests. 

Furthermore, if in the future there is more update actions for enums, they already have its own class